marlowe-dashboard-client/src/Contract/View.purs

```diff
@@ -0,0 +1,23 @@
+module Contract.View
+  ( renderContractSetup
+  , renderContractDetails
+  ) where
+
+import Prelude hiding (div)
+import Effect.Aff.Class (class MonadAff)
+import Halogen (ComponentHTML, ClassName(ClassName))
+import Halogen.HTML (div, text)
+import Halogen.HTML.Properties (classes)
+import MainFrame.Types (Action, ChildSlots, Contract, ContractTemplate)
+
+renderContractSetup :: forall m. MonadAff m => ContractTemplate -> ComponentHTML Action ChildSlots m
```

Comment:
Good point, I guess we don't know yet what will need to be a component. Hopefully nothing.

I quite like the `render` prefix, so that it's obvious it's a view function, and so you avoid potential name clashes (e.g. `renderContracts` takes `contracts` as an argument).